src/os/bluestore/BlueFS.cc

```diff
@@ -18,6 +18,19 @@
 #undef dout_prefix
 #define dout_prefix *_dout << "bluefs "
 using TOPNSPC::common::cmd_getval;
+/*
+  dout priority hierarchy
+  1 - initializations in/out, any critical error
+  2 - initialization results
+  3 - allocation failures
+  5 - init/cleanup logs
+  8 - gifting/reclaiming BlueFS space
+ 10 - RocksDB API in/out
+ 15 - RocksDB API results/errors
```

Comment:
Perhaps TOO-O-O high for errors. And perhaps for results too.  I'd prefer to have errors at derr and results at 10 (unless it's too verbose in the real life)